<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAddCar</class>
 <widget class="QDialog" name="DAddCar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addCar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Марка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Мощность</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Двигатель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Год</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Кузов</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineMake"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineModel"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePower"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEngine"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineYear"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePlat"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>linePower</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DAddCar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DAddCar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
